godot_project/autoloads/about_msep_one/about_msep_attribution_page.gd

```diff
@@ -0,0 +1,112 @@
+class_name AboutMsepAttributionPage extends MarginContainer
+
+var _tree: Tree
+var _rich_text_label: RichTextLabel
+
+var _last_selected_item: TreeItem = null
+var _custom_control: Control = null
+
+func _notification(what: int) -> void:
+	if what == NOTIFICATION_SCENE_INSTANTIATED:
+		_tree = %Tree as Tree
+		_rich_text_label = %RichTextLabel as RichTextLabel
+		
+		_tree.clear()
+		var _root: TreeItem = _tree.create_item()
+		_tree.hide_root = true
+	if what == NOTIFICATION_READY:
+		_tree.item_selected.connect(_on_tree_item_selected)
+		_rich_text_label.meta_clicked.connect(_on_rich_text_label_meta_clicked)
+
+
+func _create_software_tree_item(in_info: LicenseInfo) -> TreeItem:
+	var tree_item: TreeItem = _tree.create_item(_tree.get_root())
+	const COLUMN_0: int = 0
```

Comment:
nit: I can see this constant is used across many functions, probably could be moved to the class directly